src/domexpression.js

```diff
@@ -306,10 +306,10 @@ var _ = Mavo.DOMExpression = $.Class({
 				this.vars[name] = {
 					observe: function() {
 						this.callback = this.callback || _.special.update.bind(this);
-						target.addEventListener(type, this.callback);
+						$.bind(target, type, this.callback);
 					},
 					unobserve: function() {
-						target.removeEventListener(type, this.callback);
+						$.unbind(target, { [type]: this.callback });
```

Comment:
Doesn't `unbind()` also have a `(target, type, callback)` signature?

Reply:
Oops. It has, for sure. Fixed. Thank you!